src/client/components/Forms/SignUp.js

```diff
@@ -10,7 +10,7 @@ export default function SignUp({ onSubmit }) {
   const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
   const handleSubmit = (e) => {
     e.preventDefault();
-    onSubmit({ email, password, passwordConfirm });
+    onSubmit({ email, password, passwordConfirm, fullName: 'Jane Doe' });
```

Comment:
What is the reason why we have a "dummy name" is that just for the v1? and we will replace it later with a name field?

Reply:
its a default value to be able to test the user and create it in firebase